addons/l10n_latam_check/wizards/l10n_latam_payment_mass_transfer.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, api, fields, _
+from odoo.exceptions import UserError
+
+
+class L10nLatamPaymentMassTransfer(models.TransientModel):
+    _name = 'l10n_latam.payment.mass.transfer'
+    _description = 'Checks Mass Transfers'
+
+    payment_date = fields.Date(
+        string="Payment Date",
+        required=True,
+        default=fields.Date.context_today,
+    )
+    destination_journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        string='Destination Journal',
+        domain="[('type', 'in', ('bank', 'cash')), ('company_id', '=', company_id), ('id', '!=', journal_id)]",
+    )
+    company_id = fields.Many2one(
+        related='journal_id.company_id',
+    )
+    communication = fields.Char(
+        string="Memo",
+    )
+    journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        readonly=False,
+    )
```

Comment:
new field payment_ids with default_value
journal_id = fields.Many2one(related='payment_ids.journal_id')
re-use payment_ids field later instead of using "self._context.get('active_ids"

@zaoral

Reply:
No need to have the field `journal_id` actually